test/libsolidity/smtCheckerTests/special/tx_data_immutable_fail.sol

```diff
@@ -0,0 +1,61 @@
+pragma experimental SMTChecker;
+
+contract C {
+	bytes32 bhash;
+	address coin;
+	uint dif;
+	uint glimit;
+	uint number;
+	uint tstamp;
+	bytes mdata;
+	address sender;
+	bytes4 sig;
+	uint value;
+	uint gprice;
+	address origin;
+
+	function f() public payable {
+		bhash = blockhash(12);
+		coin = block.coinbase;
+		dif = block.difficulty;
+		glimit = block.gaslimit;
+		number = block.number;
+		tstamp = block.timestamp;
+		mdata = msg.data;
+		sender = msg.sender;
+		sig = msg.sig;
+		value = msg.value;
+		gprice = tx.gasprice;
+		origin = tx.origin;
+
+		fi();
+
+		assert(bhash == blockhash(122));
+		assert(coin != block.coinbase);
```

Comment:
Shouldn't there be some expectation to check in comments at the end of this test? Or does `isoltest` infer this looking at the `_fail` suffix in the file name?

And if that's the case then just one failed assertion will make the test pass so wrong behavior of others will not be detected.

Reply:
I had forgotten to update the expectation locally, it has them now.